### PR TITLE
Changed links for Imprint and Privacy Policy

### PR DESCRIPTION
**Pull request by sami1riaz**

### CODE REVIEW

**Comment by sami1riaz** [comment]
Do we need to change the link for Github as well? There's a TODO in Footer.tsx to change it but I don't know what we want to change it to. This maybe: https://github.com/pledge4future ?

**Comment by redfrexx** [comment]
> Do we need to change the link for Github as well? There's a TODO in Footer.tsx to change it but I don't know what we want to change it to. This maybe: https://github.com/pledge4future ?

yes true, makes sense.

**Comment by sami1riaz** [comment]
Looks like that was a very old component and the layout components weren't being used at all so I've removed them. Will leave the Github link to this repo for now.

File: frontend/src/views/App/AppFooter.tsx [hunk 1]
```diff
@@ -156,24 +151,18 @@ export default function AppFooter() {
                 <Link href="/contact">Contact</Link>
               </li>
               <li className={classes.listItem}>
-                <Link href="/impressum">Impressum</Link>
+                <Link href="https://heigit.org/imprint/">Impressum</Link>
```
Comment (redfrexx) [inline frontend/src/views/App/AppFooter.tsx:154]: I think "Impressum" is the German word, "imprint" would be the correct word in English right?

Comment (codingfabi) [reply frontend/src/views/App/AppFooter.tsx:154]: Suggestion: Change the text of the link to "imprint" aswell :)

**Pull request merged**